client/src/components/DataFiles/DataFilesProjectsList/DataFilesProjectsList.js

```diff
@@ -116,7 +121,9 @@ const DataFilesProjectsList = ({ modal }) => {
       contentShouldScroll
       manualContent
     >
-      {!modal && <DataFilesProjectsSearchbar />}
+      {!modal && sharedWorkspaces && !sharedWorkspaces.hideSearchBar && (
+        <DataFilesProjectsSearchbar />
```

Comment:
We don't want to hide this Searchbar, this is to search for projects. We only want to hide the searchbar within a project, that searches a given project's data